freqtrade/freqtradebot.py

```diff
@@ -579,6 +579,25 @@ def get_real_amount(self, trade: Trade, order: Dict) -> float:
                         f"(from {order_amount} to {real_amount}) from Trades")
         return real_amount
 
+    def get_sell_rate(self, pair: str, refresh: bool) -> float:
+        """
+        Get sell rate - either using get-ticker bid or first bid based on orderbook
+        The orderbook portion is only used for rpc messaging, which would otherwise fail
+        for BitMex (has no bid/ask in get_ticker)
+        or remain static in any other case since it's not updating.
+        :return: Bid rate
+        """
+        config_ask_strategy = self.config.get('ask_strategy', {})
+        if config_ask_strategy.get('use_order_book', False):
+            logger.debug('Using order book to get sell rate')
+
+            order_book = self.exchange.get_order_book(pair, 1)
+            rate = order_book['asks'][0][0]
```

Comment:
Ok, this is just a refactoring (sell rate fetch wise). but I think we have to convert that "asks" to "bids". but we can do it in a separate PR ... or here ... either way.

Reply:
Good point!